src/pages/ecosystem/resources/past-seminars.js

```diff
@@ -12,27 +17,37 @@ export default function PastSeminars() {
         description="Blockchain Past Webinars &amp; Seminars. Access to all our content. Substrate blockchain enables developers to quickly and easily build future-proof blockchains optimized for any use case. "
       />
       <Section>
-        <h1 className="text-4xl font-bold font-body mb-6">Blockchain Past Webinars &amp; Seminars</h1>
+        <h1 className="text-4xl font-bold font-body mb-6">Past Seminars</h1>
+      </Section>
+      <Section>
+        <h2 className="text-2xl font-normal font-body mb-6 ">Substrate Seminar Live Streams (2023)</h2>
+        <div className="flex flex-wrap gap-8">
+          <iframe
+            className="w-full lg:w-5/12 seminar-video rounded shadow-xl hover:scale-105 transition-transform"
+            src="https://www.youtube-nocookie.com/embed/videoseries?list=PLOyWqupZ-WGsfgxkwTdMOwnbRW4nx_T-i"
+            title="YouTube video player"
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowFullScreen
+          ></iframe>
+        </div>
       </Section>
-
       <Section>
-        <div className="grid sm:grid-cols-2 xl:grid-cols-3 gap-6 auto-rows-fr">
-          {seminars.map(({ id, title, date }, idx) => (
-            <Link to={`https://www.youtube.com/watch?v=${id}`} key={idx}>
-              <div className="h-full shadow-xl p-8 rounded-md bg-white dark:bg-substrateBlackish flex flex-col justify-between hover:scale-105 transition-transform">
-                <h3 className="text-xl font-bold mb-4">{title}</h3>
-                <div>
-                  <div className="flex mb-4">
-                    <Icon className="mr-2 mt-0.5 fill-current text-black dark:text-white" name="date" />
-                    <p className="mb-0">{date}</p>
-                  </div>
-                  <LineArrowButton link={`https://www.youtube.com/watch?v=${id}`} primary>
-                    Watch
-                  </LineArrowButton>
-                </div>
-              </div>
-            </Link>
-          ))}
+        <h2 className="text-2xl font-normal font-body mb-6">Substrate Seminar Live Streams (2021-2022)</h2>
+        <div className="flex flex-wrap gap-8">
+          {oldPlaylists.map((playlist, idx) => {
+            return (
+              <iframe
+                key={idx}
```

Comment:
FYI, not a showstopper.. but prob best to use the video IDs as the react key.. as not really the 'done' thing to use `index` from the array in prod.
The YT IDs will always be unique.. so fine to use these.